layers/+vim/evil-better-jumper/README.org

```diff
@@ -0,0 +1,31 @@
+#+TITLE: evil-better-jumper layer
+#+TAGS: layer|vim
+
+* Table of Contents                                       :TOC_4_gh:noexport:
+- [[#description][Description]]
+  - [[#features][Features:]]
+- [[#install][Install]]
+- [[#key-bindings][Key bindings]]
+
+* Description
+
+This layer adds support for [[https://github.com/gilbertw1/better-jumper][better-jumper]]. A configurable jump list
```

Comment:
Optional:
A short description (one or two sentences) of how this package differentiate itself from vanilla evil-jump would be helpful.